docs/source/releases.rst

```diff
@@ -1,13 +1,28 @@
-=================
- Release History
+Release History
+###############
+
+v0.14.0 (2023-10-16)
+====================
+
+Features
+--------
+- Adds :any:`WeakPartialMethodSlot`, which handles cleanup for partial methods
+  used as qt slots (callbacks).
+- Adds :any:`PydmDemotionFilter`, which makes Pydm-based applications with logging
+  configurations less verbose at close when there are cleanup issues.
+
+Contributors
+------------
+- tangkong
+- zllentz
 
 
 v0.13.0 (2023-09-27)
 ====================
 
 Features
 --------
-- Add cleaned up and performant version of heavily-used ``get_info`` in
+- Add cleaned up and performant version of heavily-used :any:`get_info` in
```

Comment:
The `:any:` cross-reference is new to me, it's a nice fallback if we don't want to specify `:class:` or something.

Reply:
Not only does it save you from specifying, it saves you from getting the whole path to the object, so you don't need to do
```
:class:`pcdsutils.modulename.classname`
```